docs/source/en/tasks/video_classification.mdx

```diff
@@ -127,7 +127,12 @@ Next up, you will derive the set of labels present in the dataset. Also, create
 * `id2label`: maps the integers to class names. 
 
 ```py 
->>> class_labels = sorted({str(path).split("/")[2] for path in all_video_file_paths})
+>>> from pathlib import Path
+
+>>> base_path = Path("/path/to/UCF101_subset")
```

Comment:
Should reuse the `file_path` above here.

Reply:
well spotted